backend/src/api/routes/user/user.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
+/* eslint-disable promise/no-callback-in-promise */
+import express from 'express';
+import {
+  body, param,
+} from 'express-validator';
+// eslint-disable-next-line import/extensions
+import { validate } from '../../validation/validate.js';
+// import controller from '../../controller/user/userController.js';
+
+import {
+  createUser, getUsers, getUser, updateUser, deleteUser,
+} from '../../controller/user/userController.js';
+
+const router = express.Router();
+
+router.post('/', validate([
+  body('firstName').notEmpty().withMessage('Firstname is empty'),
+  body('lastName').notEmpty().withMessage('Lastname is empty'),
+  body('email').notEmpty().isEmail().withMessage('Email is not valid'),
```

Comment:
nên tách ra 2 withMessage để dễ track lôi
`notEmpty().withMessage('Email is not empty'),isEmail().withMessage('Email is not valid'),`